pkg/test/testPackages.proj

```diff
@@ -155,13 +168,13 @@
   <Target Name="RestoreProjects"
           DependsOnTargets="GenerateProjects">
     <Message Importance="High" Text="*** Restoring ***" />
-    <Exec Command="$(TestRestoreCommand) &quot;$(TestProject)&quot;" StandardOutputImportance="High" />
+    <Exec Command="$(TestRestoreCommand) &quot;$(TestProject)&quot;" EnvironmentVariables="@(CliEnvironment)" ContinueOnError="ErrorAndContinue" StandardOutputImportance="High" />
```

Comment:
Given that these two are now ErrorAndContinue, should we add an `<Error ...` task to check if there where any errors so that we provide a message that is more clear? For example we  could say here that the part that failed was restore, and in the other one we could say that it was during test execution.